docs/details/dashboards/dashboard-postgresql-vacuum-monitoring-experimental.md

```diff
@@ -0,0 +1,24 @@
+# Experimental PostgreSQL Vacuum Monitoring
+
+!!! caution alert alert-warning "Disclaimer"
+    This is an Experimental Dashboard that is not part of the official Percona Monitoring and Management (PMM) deployment and might be updated. We ship this Dashboard to obtain feedback from our users.
+
+
+![!image](../../_images/PMM_PostgreSQL_vacuum_monitor-experimental.png)
+
+
+This dashbaord provides timely insights into the autovacuum process in PostgreSQL.
+
+This dashboard contains the following:
+
+- Dead tuples - Identifies the number of dead rows in each table even though the rows are physically removed from the table.
+
+- Last time vacuum ran - Tracks the last time a vacuum or autovacuum process successfully ran on each of your tables.
+
```

Comment:
I couldnt find the last 3 metrics/data on the dashboard. Can you check it again?

Reply:
@JiriCtvrtka Could you please confirm if we capture these metrics?

Reply:
Verified with Anton and we have these metrics even after last required changes from DGB.

Running instance with latest changes:

http://3.140.254.197/graph/d/postgres_vacuum_monitoring/postgresql-vacuum-monitoring?orgId=1&refresh=1m

Reply:
Thanks @JiriCtvrtka for confirming.